src/Symfony/Component/Messenger/Bridge/Amqp/Tests/Transport/ConnectionTest.php

```diff
@@ -266,7 +266,7 @@ public function testItSetupsTheConnectionWithDefaults()
         );
 
         $amqpExchange->expects($this->once())->method('declareExchange');
-        $amqpExchange->expects($this->once())->method('publish')->with('body', null, AMQP_NOPARAM, ['headers' => [], 'delivery_mode' => 2]);
+        $amqpExchange->expects($this->once())->method('publish')->with('body', null, AMQP_NOPARAM, ['headers' => [], 'delivery_mode' => 2, 'timestamp' => time()]);
```

Comment:
I agree with this feature. But the test is now time-sensitive, see https://symfony.com/blog/new-in-symfony-2-8-clock-mocking-and-time-sensitive-tests
@nicolas-grekas is it enough to mark the test as time-sensitive to make sure time() returns the same value when called in multiple places?

Reply:
fixed